cmd/main.go

```diff
@@ -4,24 +4,43 @@ import (
 	"bufio"
 	"flag"
 	"fmt"
-	"log"
 	"os"
 	"time"
 
 	"github.com/kevmo314/appendable/pkg/appendable"
+	"go.uber.org/zap"
```

Comment:
Ah, can we use slog? It was added in the latest version of golang https://go.dev/blog/slog, one fewer dependency.

Reply:
Ok